BIPs/2025-W41/BIP-875-create_ezkl_poolset.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1759342022116,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.18.2",
+    "createdFromSafeAddress": "0x9ff471F9f98F42E5151C7855fD1b5aa906b1AF7e",
+    "createdFromOwnerAddress": "",
+    "checksum": "0xdcd899bb90598ff1911d749a19732122431406be563cf340f3d534cef009814b"
+  },
+  "transactions": [
+    {
+      "to": "0x8A8B9f35765899B3a0291700141470D79EA2eA88",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "address",
+            "name": "initialManager",
+            "type": "address"
+          },
+          {
+            "internalType": "bytes32[]",
+            "name": "newPoolIds",
+            "type": "bytes32[]"
+          }
+        ],
+        "name": "createPoolSet",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "initialManager": "0xB7aadD330A64088A85e500874DCDcFB7F253fEB4",
+        "newPoolIds": "[0x3de27efa2f1aa663ae5d458857e731c129069f29000200000000000000000588]"
```

Comment:
This is wstETH / AAVE in V2 from weighted pool V4: https://balancer.fi/pools/ethereum/v2/0x3de27efa2f1aa663ae5d458857e731c129069f29000200000000000000000588 ✔️